todo_app/data/db_items.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pymongo
+from todo_app.data.item import Item
+
+def get_items():
+    client = pymongo.MongoClient(os.getenv('COSMOSDB_PRIMARY_CONNECTION_STRING'))
+    collection = client.todo_db.todo_items
+    items = []
+    for row in collection.find():
+        print("reading row")
+        items.append(row)
+        print(row)
+    return Item.from_db_items(items)
+
+def add_item(new_item):
+    client = pymongo.MongoClient(os.getenv('COSMOSDB_PRIMARY_CONNECTION_STRING'))
+    collection = client.todo_db.todo_items
+    new_item = {
+        "name": new_item,
+        "status": "To Do"
+    }
+
+    collection.insert_one(new_item).inserted_id
+
+def move_item(id, new_list):
+    client = pymongo.MongoClient(os.getenv('COSMOSDB_PRIMARY_CONNECTION_STRING'))
+    collection = client.todo_db.todo_items
+    collection.update_one(
+        {"_id": id}, {"$set": {"status": new_list}}
```

Comment:
This doesn't work for me - clicking the "Doing" etc. buttons on the webpage does nothing - and I think that's because here you've got a type mismatch; `_id` is an `ObjectId` (e.g. if you look in the Data Explorer you'll see `ObjectId('abcsdsad')` so we need to convert `id` to the same. We can do that with the ObjectId type imported from `bson` (which is already included as part of pymongo), and then convert as below:
```suggestion
        {"_id": ObjectId(id)}, {"$set": {"status": new_list}}
```